cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -0,0 +1,38 @@
+
+{# ==========================================================================
+
+   table.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a gray well organism.
+   See [GHE]/flapjack/Modules-V1/wiki/Well
+
+   table.row
+
+
+   ========================================================================== #}
+
+{% macro render(thead, rows) %}
+<table class="table__stack-on-small table__entry-header-on-small">
```

Comment:
There was a thread on cf-typography about `list` (which has no styles), but I believe Mike and Scott said still needs to be included. This is an analogous use-case. Maybe @Scotchester remembers the context?